README.md

```diff
@@ -24,6 +24,49 @@ Include it in your Cargo.toml:
 inotify = "*"
 ```
 
+And here's a little example:
+```Rust
+#![feature(path)]
+
+extern crate inotify;
+
+use inotify::INotify;
+use inotify::ffi::*;
+use std::path::Path;
+
+fn main() {
+    let mut ino = INotify::init().unwrap();
+
+    ino.add_watch(Path::new("/home"), IN_MODIFY | IN_CREATE | IN_DELETE).unwrap();
+    loop {
+        let events = ino.wait_for_events().unwrap();
+
+        for event in events.iter() {
+            if event.mask & IN_CREATE > 0 {
```

Comment:
I only converted C code to Rust but these functions are very nice ! I change that.